requirements.txt

```diff
@@ -8,3 +8,4 @@ protobuf==3.6.1
 python-magic==0.4.15
 six==1.12.0
 Werkzeug==0.14.1
+python-libmagic==0.4.0
```

Comment:
Two different libraries. `python-magic==0.4.15` should suffice.

Reply:
I've had to use `python-magic-bin==0.4.14` to prevent this issue.